packages/calling-component-bindings/src/captionsSelector.ts

```diff
@@ -122,6 +122,16 @@ export const _captionsBannerSelector: _CaptionsBannerSelector = reselect.createS
     // Following Teams app logic, no matter how many 'Partial' captions come,
     // we only pick first one according to start time, and all the other partial captions will be filtered out
     // This will give customers a stable captions experience when others talking over the dominant speaker
+    // First turn all partial captions that are older than 10 seconds to final
+    captions
+      ?.filter((captions) => captions.resultType === 'Partial')
+      .forEach((c) => {
+        // if c is created more than 15 seconds ago, make it final
+        if (c.timestamp.getTime() + 15000 < Date.now()) {
```

Comment:
15seconds feels long for the somewhat broken experience. Does a shorter time work? 5seconds?

Reply:
Yep updated to 5

Reply:
Thanks for checking up with Teams on this Caroline! :D